cpp/benchmarks/flow_simulation_ode_secirvvs.h

```diff
@@ -64,8 +64,7 @@ class FlowlessModel : public CompartmentalModel<osecirvvs::InfectionState,
         auto icu_occupancy           = 0.0;
         auto test_and_trace_required = 0.0;
         for (auto i = AgeGroup(0); i < n_agegroups; ++i) {
-            auto rateINS =
-                0.5 / (params.get<osecirvvs::IncubationTime>()[i] - params.get<osecirvvs::SerialInterval>()[i]);
+            auto rateINS = 1 / params.get<osecirvvs::TimeInfectedNoSymptoms>()[i];
```

Comment:
Consider using 1 / params.get<osecirvvs::TimeInfectedNoSymptoms>()[i] directly in the formula